CHANGELOG.md

```diff
@@ -2,7 +2,7 @@ CHANGELOG
 =========
 
 ## HEAD (Unreleased)
-_(none)_
+* Default Lambda NodeJS usage to Node JS 10.x
```

Comment:
I would mention that existing lambdas will stay on whatever version they were created with.  Users will need to explicitly move the lambdas to 10x if they want that version.